Source/Search/Mixins/AdvancedSearchProviderMixin.lua

```diff
@@ -0,0 +1,143 @@
+local _, addonTable = ...
+local zc = addonTable.zc
```

Comment:
Just because `zc` is cryptic, and we only need the `StringContains`, let's move `StringContains` into `Auctionator.Utilities`.